pkgs/by-name/pr/prettier/package.nix

```diff
@@ -103,7 +103,7 @@ let
   nodeEntryPointOf =
     plugin:
     let
-      pluginDir = "${plugin.outPath}/lib/node_modules/${plugin.packageName}";
+      pluginDir = "${plugin.outPath}/lib/node_modules/${plugin.pname}";
```

Comment:
Other references, and doc-comments, for `packageName` may need updated to `pname` too to avoid future issues being opened when `lib.warn` and/or `throw` branches get tripped

Reply:
Done!